src/utils.js

```diff
@@ -3,109 +3,101 @@
  */
 
 // Constants
-const cssPrefixes = ['-moz-', '-ms-', '-o-', '-webkit-'];
+const cssPrefixes = 'moz ms o webkit'.split(' ');
```

Comment:
``style.webkitProp`` instead of ``style['-webkit-Prop']``

Reply:
Are you sure? I mean, the webkit props (see chrome dev console `>` computed) are all prefixed with `-webkit-`...

Reply:
hmm, not 100% sure, I could totally be wrong. But, from what I experienced setting ``element.style['-webkkit-Prop'] = "something"`` doesn't seem to work. Apologies if I messed stuff up.

Reply:
Oh, your right (https://stackoverflow.com/questions/708895/how-to-set-the-style-webkit-transform-dynamically-using-javascript)...

Wow, thought it would be simply adding a prop which starts with this prefix... didn't know that ^^

Reply:
No worries. Everyone makes mistakes.